edb/api/errors.txt

```diff
@@ -138,6 +138,9 @@
 
 0x_FF_00_00_00   ClientError
 0x_FF_01_00_00   ClientConnectionError
+0x_FF_01_01_00   ConnectionFailedError
+0x_FF_01_01_01   ConnectionFailedTemporarilyError
+0x_FF_01_02_00   ConnectionTimeoutError
```

Comment:
```suggestion
0x_FF_01_01_00   ClientConnectionFailedError
0x_FF_01_01_01   ClientConnectionFailedTemporarilyError
0x_FF_01_02_00   ClientConnectionTimeoutError
```

Reply:
What other connection could fail? I don't think we expose any server-side connection errors with similar granularity. Keeping length of class names (including errors) sane is useful. I would rather `ClientConnectionError -> ConnectionError`.

Reply:
The `Client` prefix is customary for errors generated by the _client_.

> What other connection could fail? I don't think we expose any server-side connection errors with similar granularity. 

Not yet, but it's easy to imagine connection errors stemming from server-side API integrations and foreign data wrappers.